.changeset/violet-mugs-tickle.md

```diff
@@ -0,0 +1,5 @@
+---
+"@logto/schemas": minor
+---
+
+Add `type` field to `roles` schema.
```

Comment:
I think changeset should be some specific fix, or briefly describe a feature. This is too simplified.